tests/unit/models/test_reservation_model.py

```diff
@@ -6,38 +6,34 @@ def test_reservation_creation(tables, new_reservation):
     new_reservation.save()
 
 
-def test_interference_detection(tables, new_reservation):
+def test_interfering_reservation_cannot_be_saved(tables, new_reservation, new_reservation_2):
```

Comment:
Yep, I changed this, because the test was failing. No exception was thrown by would_interfere in models/Reservation.py because the two reservations had the same ID. Filters were added to would_interfere in #754771c898df88dc5d8f38fc7b658b5fe368646c that broke the tests, but nobody checked this until now. That's why I'm trying to push the travis pytests and linter.